src/components/containers/ModalProvider.js

```diff
@@ -73,10 +73,10 @@ class ModalProvider extends React.Component {
   render() {
     const {component: Component, componentProps, isAnimatingOut} = this.state;
     return (
-      <Fragment>
+      <>
```

Comment:
oh wow..

Reply:
yep, that's about the only benefit we get from babel 7 😛

Reply:
lol, I kind of liked it better when it was called Fragment and we saw where it was imported from, just thinking of devs new to React who see all those weird little syntaxes, that show little clue as to where they're coming from..
but oh well.. that's how it is I guess